squat_analysis/function/handler.py

```diff
@@ -178,13 +141,14 @@ def handler(event, context):
     y_axes.remove('mtp_angle_r')
     y_axes.remove('mtp_angle_l')
     
-    # Create results dictionnary.
+    # Create results dictionary.
     results = {
         'indices': times, 
         'metrics': squat_scalars, 
         'datasets': datasets,
         'x_axis': 'time', 
-        'y_axis': y_axes}
+        'y_axis': y_axes,
+        'squat_type': squat_type}
```

Comment:
Are you planning to change the dashboard to include this info?

Reply:
yes, i was planning on adding the detected type and seeing if sam would like it. otherwise, can leave it off for now and add it later if needed